src/catalogue/catalogue.component.test.tsx

```diff
@@ -122,7 +122,7 @@ describe('Catalogue', () => {
     createView('/catalogue/4');
 
     await waitFor(() => {
-      expect(screen.getByText('Description')).toBeInTheDocument();
+      expect(screen.getByText('Last modified')).toBeInTheDocument();
```

Comment:
```suggestion
      expect(screen.getByText('Description')).toBeInTheDocument();
```
Can this go back to description since `Last modified` appears in the cards as well.

Reply:
It doesn't pass with `Description`. Which is strange as I tried using `Cameras 1` - which would be the top in the table view and it passes. Not sure what is going on there. Is `Cameras 1` okay to use?

Reply:
Ah right, yes I saved the output to a file `DEBUG_PRINT_LIMIT=100000 yarn test catalogue.component.test.tsx >> test.html 2>&1` and it only renders up to View Items because of the virtualisation. `Cameras 1` is fine to use, as is `View Items` I think.